packages/components/src/components/navmenuitem/navmenuitem.styles.ts

```diff
@@ -138,18 +153,30 @@ const styles = [
       flex-shrink: 0;
     }
 
-    :host(:dir(ltr))::part(arrow) {
+    :host([show-label]:dir(ltr))::part(arrow) {
       position: absolute;
       right: -0.75rem;
       top: 0.875rem;
     }
 
-    :host(:dir(rtl))::part(arrow) {
+    :host([show-label]:dir(rtl))::part(arrow) {
       position: absolute;
       left: -0.75rem;
       top: 0.875rem;
     }
 
+    :host(:dir(ltr))::part(arrow) {
+      position: absolute;
+      right: -0.25rem;
+      top: 0.875rem;
+    }
+
+    :host(:dir(rtl))::part(arrow) {
+      position: absolute;
+      left: -0.25rem;
+      top: 0.875rem;
+    }
+
```

Comment:
Please use logical values instead:

```suggestion
    :host::part(arrow) {
      position: absolute;
      inset-inline-end: -0.25rem;
      top: 0.875rem;
    }
```